code/modules/mob/living/simple_animal/abnormality/waw/naked_nest.dm

```diff
@@ -76,6 +76,14 @@
 		serpentsnested = serpentsnested + 1
 	return
 
+/mob/living/simple_animal/hostile/abnormality/naked_nest/death(gibbed)
+	for(var/atom/movable/AM in src)
+		AM.forceMove(loc)
```

Comment:
It's prefered that you use get_turf(src) instead of loc, since loc can be other object at times.